Assets/GothicVR/Scripts/Phoenix/Util/CheckIfAudible.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace GVR.Phoenix.Util
+{
+    public class CheckIfAudible : MonoBehaviour
```

Comment:
Please: rename it so that it reflects the use case better. Like: _InaudibleSoundDisabler_